README.md

```diff
@@ -84,6 +84,7 @@
 - [TorBirdy](https://www.torproject.org/getinvolved/volunteer.html.en#project-torbirdy) - Torbutton for Thunderbird and related Mozilla mail clients.
 - [TorCheck](https://www.torproject.org/getinvolved/volunteer.html.en#project-torcheck) - Site for determining if the visitor is using Tor or not.
 - [multitor](https://github.com/trimstray/multitor) - Shell scripts to automate creation of multiple Tor instances, load-balanced with HAProxy.
+- [aria2-onion-downloader](https://github.com/sn0b4ll/aria2-onion-downloader) - Download from .onion-domains more quickly.
```

Comment:
Please:

* alphabetize,
* use Markdown formatting for technical content,
* don't hyphenate words that are not compound words,
* provide clarity and context in the short description.

```suggestion
- [aria2-onion-downloader](https://github.com/sn0b4ll/aria2-onion-downloader) - Docker Compose configuration to massively speed up large downloads published to `.onion` domains, such as gigabyte-sized leaks split into multiple parts.
- [multitor](https://github.com/trimstray/multitor) - Shell scripts to automate creation of multiple Tor instances, load-balanced with HAProxy.
```